standard-cluster-services/src/main/java/org/terracotta/monitoring/IMonitoringProducer.java

```diff
@@ -0,0 +1,31 @@
+package org.terracotta.monitoring;
+
+import com.tc.classloader.CommonComponent;
+
+
```

Comment:
When we add the statistics based stuff (ring buffer) will that have a completely separate interface?  If so we might want to use a more specific name for this interface